node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
this file shouldn't be committed

Reply:
I wonder how it got committed, I will remove it.